themes/theme-default/src/shared/options.ts

```diff
@@ -192,18 +192,14 @@ export interface DefaultThemeLocaleData extends LocaleData {
   /**
    * Sidebar depth
    *
-   * - Set to `0` to disable all levels
-   * - Set to `1` to include `<h2>`
-   * - Set to `2` to include `<h2>` and `<h3>`
-   * - ...
+   * - `false`: No headers.
+   * - `number`: only headings of that level will be displayed.
```

Comment:
I disagree with this change, there is no meaning to display, e.g., only h4 header, the number behavior should behave like the previous.